### PR TITLE
Further refinement of the outreachy landing page

### PR DESCRIPTION
**Pull request by objectiser**
Signed-off-by: Gary Brown <gary@brownuk.com>

### CODE REVIEW

**Comment by objectiser** [comment]
@jpkrohling @yurishkuro @pavolloffay @bhs Updates applied based on comments from #224. Let me know if I have missed anything, or if changes required.

File: pages/outreachy.md [hunk 1]
```diff
@@ -69,42 +69,90 @@ tracer for mobile usage
...
 * Clear reported spans between tests. Automatically assert on errors.
 
+Desired skills: Java
+
 ### Move community Jaeger tracers to the jaegertracing organization.
 
 Mentor: Gary Brown
 
-* This task include writing xdock tests, missing reporters..
+This task include writing xdock tests, missing reporters. Candidates include:
```
Comment (yurishkuro) [inline pages/outreachy.md]: S/xdock/integration

**Comment by yurishkuro** [comment]
Can we consider putting OT and Jaeger projects into clearly separated sections? Or even moving Jaeger projects to Jaeger website to keep the ot site vendor neutral? Our we could move the whole page to Jaeger website, including OT parts and just link to it from ot site.

**Comment by objectiser** [comment]
Some of the tasks might span across projects - but I understand your point. Wonder if it would be possible to host on the cncf website, as they are sponsoring it?
cc @caniszczyk

**Comment by jpkrohling** [comment]
> Wonder if it would be possible to host on the cncf website, as they are sponsoring it?

For this program (Dec-Mar), I'd keep it on OT's website, as it has been publicized already: https://www.outreachy.org/apply/rounds/2017-december-march/

For the next iteration, I'd prefer to have it on Jaeger's website instead.

**Comment by reimertz** [comment]
LGTM, will merge!

**Pull request merged**